web/templates/topic_edit_form.hbs

```diff
@@ -1,10 +1,9 @@
 {{! Client-side Handlebars template for rendering the topic edit form. }}
 
 <form id="topic_edit_form">
```

Comment:
Hmm, this should be a class, since there can be more than one copy of this element in the DOM at once.

Reply:
Makes sense, updated.